provider/core/src/data_provider.rs

```diff
@@ -324,6 +324,23 @@ where
         })
     }
 
+    pub fn from_rc_buffer_yoke(yoke: Yoke<M::Yokeable, Rc<[u8]>>) -> Self {
+        Self {
+            inner: DataPayloadInner::RcBuf(yoke)
+        }
+    }
+
+    pub fn try_from_yoked_buffer<T, E>(
```

Comment:
issue: docs

Reply:
(and elsewhere)

no need for examples, just useful to have a one-liner about what's going on